bip39-create-accounts/create-accounts.js

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env node
+
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import readline from 'node:readline/promises';
+
+import {
+    Client,
+    AccountId,
+    PrivateKey,
+    TransferTransaction,
+    TransactionRecordQuery,
+} from '@hashgraph/sdk';
+import {
+    HDNode as ethersHdNode,
+} from '@ethersproject/hdnode';
+
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+// TODO read these in as CLI params
+const NUM_ACCOUNTS = 2;
```

Comment:
Will do in follow up PR! Note that 2 was selected to avoid the network-level entity creation rate limit. The main part of the script will need to be refactored to support a larger number.
Details: https://swirldslabs.slack.com/archives/C03HYSTK8LV/p1713920911134479?thread_ts=1713677785.249739&cid=C03HYSTK8LV